plugin/clone/src/clone_status.cc

```diff
@@ -465,6 +470,38 @@ void Status_pfs::Data::write(bool write_error) {
   status_file.close();
 }
 
+void Status_pfs::Data::write_synchronization_coordinate(
+    const Key_Value &coordinate, bool remove) {
+  std::string file_name;
+  /* Append data directory if cloning to different place. */
+  if (!is_local()) {
+    file_name.assign(m_destination);
+    file_name.append(FN_DIRSEP);
+    file_name.append(CLONE_SYNCHRONIZATION_COORDINATES_FILE);
+  } else {
+    file_name.assign(CLONE_SYNCHRONIZATION_COORDINATES_FILE);
+  }
+  if (remove) {
+    std::filesystem::remove(file_name);
```

Comment:
There could be a reason not to use `std::filesystem`: do we need to issue diagnostics in the case of file not found here? If yes, we need to print to error log, then use `my_delete` from mysys

Reply:
if file not found, it could be because it is a new instance or during version update etc, which I think is not something we particularly care about.